packages/protocol/contracts-0.8/common/EpochManager.sol

```diff
@@ -285,7 +276,7 @@ contract EpochManager is
   }
 
   function systemAlreadyInitialized() public view returns (bool) {
-    return initialized && epochManagerInitializer == address(0);
+    return initialized;
```

Comment:
We still need some flag that give us info whether initializeSystem was called. Since initialized will be set to true already after initialize will be called.

Reply:
good call. I' now checking that `elected.length >0` since this can only be set once the system is initialized.